previews.md

```diff
@@ -155,3 +206,7 @@ https://github.com/Azure/application-gateway-kubernetes-ingress
 [5]: https://docs.microsoft.com/en-us/azure/aks/cluster-autoscaler
 [6]: https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/overview
 [7]: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest
+
+[api server]: https://docs.microsoft.com/en-us/azure/aks/api-server-authorized-ip-ranges
+[windows]: https://docs.microsoft.com/en-us/azure/aks/windows-container-cli
```

Comment:
These docs are not public yet. Don't know when they will be...

Reply:
Woot, removed.